src/utils/block-helper.ts

```diff
@@ -25,19 +25,19 @@ export function getSuggetions(block: Block, fields: Array<string>) : Array<Compl
   }
 
   const blockClass = block.lookup(`.${fields[0]}`);
-  
+
   if (!blockClass) {
     return [];
   }
 
-  if (blockClass instanceof BlockClass) {
+  if (isBlockClass(blockClass)) {
     return blockClass.getAttributes().map(attr => new CompletionItem(attr.name, CompletionItemKind.Function));
```

Comment:
This getAttributes() doesn't resolve inheritance. I think we need to add a new convenience method (`resolveAttributes()`) to css blocks for this.

ditto for classes up above.